src/main/java/by/exadel/internship/service/impl/FileServiceImpl.java

```diff
@@ -45,18 +44,44 @@ private String uploadFile(byte[] file, String fileName){
         log.info("Try to upload file to cloud");
         BlobId blobId = BlobId.of(bucketName, fileName);
         BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("media").build();
-        Credentials credentials = null;
+
+        Credentials credentials = getCredentialsByJSON();
+
+        Storage storage = StorageOptions.newBuilder().setCredentials(credentials).build().getService();
+        storage.create(blobInfo, file);
+        log.info("File was uploaded to cloud");
+        return fileName;
+    }
+
+    @Override
+    public FileInfoDTO download(String fileName, String formLastName, String internshipName) {
+        log.info("Try to download file from cloud");
+        String destFileName = formLastName
+                .concat(DASH_SEPARATOR)
+                .concat(internshipName)
```

Comment:
Please rewrite with StringUtils to concat

Reply:
Fixed